views/update/update-presenter.html

```diff
@@ -1,11 +1,15 @@
 {% extends "update-layout.html" %}
 
-{% set title = "Who can we contact about this update?" %}
-
 {% block pageTitle %}
   {% include "includes/update-page-title.html" %}
 {% endblock %}
 
+{% if (journey == "remove") %}
```

Comment:
I think the title should be set before the pageTitle block like it was before this change, so can you move the pageTitle block underneath the block that is setting the title

Reply:
This makes me wonder if code is actually working, based on where this block currently sits. But yes, agree it needs to move up.

Reply:
moved block to above title block